src/Pipe/index.tsx

```diff
@@ -55,9 +55,9 @@ const PipeItem = styled<'div', { fill: keyof typeof defaultTheme.colors }>('div'
   }
 `;
 
-const Pipe: React.ComponentType<{}> & { Item: typeof PipeItem } = ({ children }) => (
-  <PipeContainer>{children}</PipeContainer>
-);
+const Pipe: React.ComponentType<React.PropsWithChildren<{}>> & { Item: typeof PipeItem } = ({
```

Comment:
while not relevant to the PR, it is not 100% clear to me what this intersection type works out as, or why it's needed.